skcosmo/sample_selection/_base.py

```diff
@@ -346,3 +397,226 @@ def __init__(
             full=full,
             random_state=random_state,
         )
+
+
+class DirectionalConvexHull:
+    """
+    Performs Sample Selection by constructing a Directional Convex Hull and determining the distance to the hull as outlined in the reference
+
+    Parameters
+    ----------
+
+    low_dim_idx    : list of ints, default None
+                   Indices of columns of X containing features to be used for the
+                   directional convex hull construction (also known as the low-
+                   dimensional (LD) hull). By default [0] is used.
+
+    Attributes
+    ----------
+
+    high_dim_idx_   : list of ints
+                    Indices of columns in data containing high-dimensional
+                    features (i.e. those not used for the convex hull
+                    construction)
+
+    vertices_idx_ : numpy.ndarray
```

Comment:
Why do we deviate from the convention of the other selectors? I.e. why is this not `selected_idx_`

Reply:
right, renamed it